couchbase/search/v1/search.proto

```diff
@@ -246,6 +246,18 @@ message Facet {
   }
 }
 
+message KnnQuery {
+  optional float boost = 1;
+  optional string field = 2;
```

Comment:
field is non-optional - you have to say what the vector field is.

(Technically the server does allow it to not be present, but then your vector query is a no-op, making that a bit useless)